src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift

```diff
@@ -129,7 +129,7 @@ enum AccessibilityIdentifiers {
 		}
 		
 		enum NotesInformation {
-			static let titel = "AppStrings.ContactDiary.NotesInformation.title"
+			static let title = "AppStrings.ContactDiary.NotesInformation.title"
```

Comment:
Have you already checked if the spelling change is not affecting where the identifier is used.

Reply:
I will check @naveeddotio.

Reply:
Mh, I have no way on an Emulator to check whether the title is still spoken when the text gets read out, sorry. 
Maybe you could check on a real device @naveeddotio?

I can also offer to revert the change on this file.

Reply:
@naveeddotio I reverted the change to this file. Please review again.